src/batch.service.ts

```diff
@@ -8,14 +8,14 @@ export class BatchService<T> extends CommonService
 	/**
 	 * fires multiple requests in parallel
 	 *
-	 * @since 4.2.0
+	 * @since 8.0.0
 	 *
-	 * @param {ObservableInput<T>[]} requestArray collection of requests
+	 * @param {ObservableInput<$>[]} requestArray collection of requests
 	 *
-	 * @return {Observable<T[]>} multiple http responses
+	 * @return {Observable<$[]>} multiple http responses
 	 */
 
-	public parallel(requestArray : ObservableInput<T>[]) : Observable<T[]>
+	public parallel<$ = T>(requestArray : ObservableInput<$>[]) : Observable<$[]>
```

Comment:
I haven't seen `$` to be used as generic type parameter.
Technically it is totally fine to me. 👍 

Another idea is, to use the generic type parameter to add documentation to the code.
 
```suggestion
	public parallel<TResponse = T>(requestArray : ObservableInput<TResponse>[]) : Observable<TResponse[]>
```